<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/logo.ico</normaloff>:/res/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{
border-image: url(:/res/bluebackground.jpg);
border-width : 1px;

border-color : #1ea4f2;

border-style : solid;

border-width : 1px;
}


*{
font-size:22px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0.8);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_word">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_query">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
border-radius:15px;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_user">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0,0,0,0);
border-radius:15px;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/user.png</normaloff>:/res/user.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_username">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background:transparent;
font-size:12px;
</string>
        </property>
        <property name="text">
         <string>小明缺的小明缺的的的</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_sent">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255,255,255,0.8);
border-radius:12px;
padding-left:10px;
padding-right:10px;
</string>
       </property>
       <property name="text">
        <string>/‘ðeəfɔ:/</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/sound2.png</normaloff>:/res/sound2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_meaning">
       <property name="styleSheet">
        <string notr="true">font-size:16px;</string>
       </property>
       <property name="text">
        <string>ad.因此，所以【绝对不可以直接连接两个独立的句子】</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0.8);
border-radius:15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_study">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);
border-radius:15px;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>学习</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_review">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 255);
border-radius:15px;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>复习</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_cet4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{     
background: rgba(0,0,0,0.4);
}
QToolButton:checked{    /*选中后*/  
background: rgb(255, 0, 0); 
}</string>
          </property>
          <property name="text">
           <string>CET4</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/bandfouricon.jpg</normaloff>:/res/bandfouricon.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>160</width>
            <height>180</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_cet6">
          <property name="styleSheet">
           <string notr="true">QToolButton{     
background: rgba(0,0,0,0.4);
}
QToolButton:checked{    /*选中后*/  
background: rgb(255, 0, 0); 
}</string>
          </property>
          <property name="text">
           <string>CET6</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/bandsixicon.jpg</normaloff>:/res/bandsixicon.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>160</width>
            <height>180</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
